### DISCUSSION TITLE
UpSnap  configuration file

### DISCUSSION THREAD

**Discussion by maxwassiljew** [seriousm4x/UpSnap#719] [Q&A]
Hi all,

is it possible to provide a configuration file where all machines and users are configured?

The background is to migrate UpSnap to another machine or in case of backup only to backup a config file?

Is this possible right now or is it planned for the future?

**Comment by jesusgt** [comment]
I'd love to have this option available too.

**Comment by seriousm4x** [comment]
I could look into importing from a config file as this is not implemented, but it's got a low priority.
Pocketbase has a backup function build into it. This can be done via the [GUI](https://pocketbase.io/docs/going-to-production/#backup-and-restore) or the [API](https://pocketbase.io/docs/api-backups/). Also, just zipping the data folder will work as well.
